From 6869c5d4b793d3d9b545bf950fac1337d15871d4 Mon Sep 17 00:00:00 2001
From: Alexandru Nemes <alexandru.nemes@enea.com>
Date: Thu, 20 Apr 2017 16:00:46 +0300
Subject: [PATCH] AArch64 support for Dovetail docker

Add aarch64 patch for Dovetail docker file;
The patch should be applied from releng

Change-Id: I1ec666012fc58a4dfdaebfde1eb7b15e089cf7ec
Signed-off-by: Alexandru Nemes <alexandru.nemes@enea.com>
---
 docker/Dockerfile               |  8 +++++---
 docker/Dockerfile.aarch64.patch | 36 ++++++++++++++++++++++++++++++++++++
 2 files changed, 41 insertions(+), 3 deletions(-)
 create mode 100644 docker/Dockerfile.aarch64.patch

diff --git a/docker/Dockerfile b/docker/Dockerfile
index 8cc15e0..bbab012 100644
--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,6 +1,6 @@
-FROM ubuntu:14.04
-MAINTAINER Leo Wang <grakiss.wanglei@huawei.com>
-LABEL version="0.1" description="OPNFV Dovetail Docker Container"
+FROM aarch64/ubuntu:16.04
+MAINTAINER Armband team <armband@enea.com>
+LABEL version="0.1" description="OPNFV Dovetail AArch64 Docker Container"
 
 RUN \
     apt-get update \
@@ -28,6 +28,8 @@ RUN \
 && \
     mkdir -p ${REPOS_DIR}/results \
 && \
+    pip install -U setuptools \
+&& \
     pip install -U pip \
 && \
     pip install -r ${REPOS_DIR}/requirements.txt \
diff --git a/docker/Dockerfile.aarch64.patch b/docker/Dockerfile.aarch64.patch
new file mode 100644
index 0000000..f3a3a23
--- /dev/null
+++ b/docker/Dockerfile.aarch64.patch
@@ -0,0 +1,36 @@
+From: Alexandru Nemes <alexandru.nemes@enea.com>
+Date: Mon, 24 Apr 2017 10:54:29 +0300
+Subject: [PATCH] Add AArch64 support for Dovetail docker file
+
+Signed-off-by: Alexandru Nemes <alexandru.nemes@enea.com>
+---
+ docker/Dockerfile | 9 ++++++---
+ 1 file changed, 6 insertions(+), 3 deletions(-)
+
+diff --git a/docker/Dockerfile b/docker/Dockerfile
+index 8cc15e0..02b41ad 100644
+--- a/docker/Dockerfile
++++ b/docker/Dockerfile
+@@ -1,6 +1,6 @@
+-FROM ubuntu:14.04
+-MAINTAINER Leo Wang <grakiss.wanglei@huawei.com>
+-LABEL version="0.1" description="OPNFV Dovetail Docker Container"
++FROM aarch64/ubuntu:16.04
++MAINTAINER Armband team <armband@enea.com>
++LABEL version="0.1" description="OPNFV Dovetail AArch64 Docker Container"
+
+ RUN \
+     apt-get update \
+@@ -28,6 +28,8 @@ RUN \
+ && \
+     mkdir -p ${REPOS_DIR}/results \
+ && \
++    pip install -U setuptools \
++&& \
+     pip install -U pip \
+ && \
+     pip install -r ${REPOS_DIR}/requirements.txt \
+@@ -37,3 +39,4 @@ RUN \
+     pip install -e .
+
+ WORKDIR ${REPOS_DIR}/dovetail
\ No newline at end of file
-- 
2.7.4

